exampleplugins/5.2/SwagProducerApi/Components/Api/Resource/Producer.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Shopware\Components\Api\Resource;
+
+use Shopware\Components\Api\Exception as ApiException;
+use Shopware\Models\Article\Supplier as SupplierModel;
+
+/**
+ * Class Producer
+ *
+ * @package Shopware\Components\Api\Resource
+ */
+class Producer extends Resource
```

Comment:
Please choose another example. We already have a resource for the `Supplier` Model, it was named manufacturer, see here: https://github.com/shopware/shopware/blob/5.2/engine/Shopware/Components/Api/Resource/Manufacturer.php#L46